src/ipolatev.f90

```diff
@@ -16,6 +25,37 @@ module ipolatev_mod
 
 contains
 
+  !> @brief Interpolates scalar fields between grids given ip_grid objects.
```

Comment:
No need to brief tag for subroutines/functions.

Reply:
Can I keep the `@details`?

Reply:
Why keep it? Is it adding something to the output? If not, then don't keep it. We would like all our doxygen to look similar, so that programmers can easily contribute doxygen to any of the NCEPLIBS codes.